File: snippets/messaging/forward-sms/forward-sms.js
// npm requirements - express, body-parser, twilio

const http = require('http');
const express = require('express');
const bodyParser = require('body-parser');

const app = express();

app.use(bodyParser.urlencoded({ extended: true }));

app.post('/forward', (req, res) => {

  const MessagingResponse = require('twilio').twiml.MessagingResponse;
  const response = new MessagingResponse();
  const message = response.message({to:'Your Phone number string here'});
  
  message.body(`From: ${req.body.From} Message: ${req.body.Body}`);

  res.send(response.toString());
});

http.createServer(app).listen(1337, () => {
  console.log('Express server listening on port 1337');
});

File: snippets/messaging/forward-sms/forward-sms.cs
public ActionResult sms()
{
var body = Request.Params.Get("body")]
var to = ConfigurationManager.AppSettings["TWILIO_ACME_CALLERID"];
var response = new MessagingResponse();
var message = new Message();
message.body(body);
message.to(to);
response.Append(message);
return Content(response.ToString(), contentType: "text/xml");
}
